apps/dashboard/src/app/team/[team_slug]/(team)/~/engine/(instance)/[engineId]/overview/components/transactions-table.tsx

```diff
@@ -532,13 +532,15 @@ export function TransactionCharts(props: {
 
   return (
     <ThirdwebBarChart
-      title="Transactions Breakdown"
-      description="Transactions sent from your backend wallets per day"
+      header={{
+        title: "Transactions Breakdown",
+        description: "Transactions sent from your backend wallets per day",
+        titleClassName: "text-xl mb-0",
```

Comment:
What makes this one special?

Reply:
Reduced the default size - so it doesn't clash with other titles on the page